.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.3.8
```

Comment:
We probably don't want both `.tool-versions` and `.ruby-version` in a single project.  And in any case... the `master` branch was upgraded to 2.5.9 since this PR was opened.  (And we don't want to downgrade prod).

It might be easiest to just remove this file again, and follow the steps in the [asdf-ruby readme](https://github.com/asdf-vm/asdf-ruby#migrating-from-another-ruby-version-manager) to make it recognize `.ruby-version` files?

Reply:
Committed with the `.tool-versions` removed.